content/app/app-dev-course/modul4/_index.en.md

```diff
@@ -57,7 +57,7 @@ The municipality of Sogndal wishes to collect information on the newcomers worki
 
 1. The municipality of Sogndal has created [a code list for **industries**](../industry.json). Download the file and place it correctly in the repository.
 
-2. Set up the values in the code list for **Years in work force** directly in the application logic.
+2. Set up the values in the code list for **Years in work force** directly in the json file of the schema.
```

Comment:
Her er det ønskelig at man gjør det programmatisk og ikke direkte i json-filen, så det vil være i C# optionsfilen hvis jeg ikke husker feil. Sjekk evt. hva lf har gjort for years in workforceverdien

Reply:
Aha, det var nettopp dette jeg ikke syntes kom så klart frem. Ronny mente også at det var json-filen som skulle endres, men jeg nå at det er C#-koden som er endret i løsningsforslaget.